boards/luatos/esp32s3_luatos_core/Kconfig

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2024 Espressif Systems (Shanghai) Co., Ltd.
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD_ESP32S3_LUATOS_CORE_EXTRA_CPU
+	bool
+	default y
+	depends on BOARD_ESP32S3_LUATOS_CORE || BOARD_ESP32S3_LUATOS_CORE_USB
+	select SOC_ESP32S3_PROCPU if BOARD_ESP32S3_LUATOS_CORE_ESP32S3_PROCPU || BOARD_ESP32S3_LUATOS_CORE_USB_ESP32S3_PROCPU
+	select SOC_ESP32S3_APPCPU if BOARD_ESP32S3_LUATOS_CORE_ESP32S3_APPCPU || BOARD_ESP32S3_LUATOS_CORE_USB_ESP32S3_APPCPU
```

Comment:
why is this selected here, and not in the `Kconfig.esp32s3_luatos_core` and `Kconfig.esp32s3_luatos_core_usb` files ?

Secondly, if you use two variants instead of two boards, then you can get rid of:
```
	bool
	default y
	depends on BOARD_ESP32S3_LUATOS_CORE || BOARD_ESP32S3_LUATOS_CORE_USB
```

Reply:
Guess this is indirectly answered here: https://github.com/zephyrproject-rtos/zephyr/pull/69194#discussion_r1507814296

Those symbols are currently not assumed to go to Kconfig.soc tree.

Though the possibility to get rid of 
```
	bool
	default y
	depends on BOARD_ESP32S3_LUATOS_CORE || BOARD_ESP32S3_LUATOS_CORE_USB
```
will still be possible if there is just one board with one variant instead of two boards.

Reply:
Fixed. luatos_core uses usb variant model now.